src/components/Game.js

```diff
@@ -0,0 +1,145 @@
+import React, {useState, useEffect, useRef } from 'react'
+import useSound from 'use-sound';
+import flipCardSound from '../sounds/card.mp3'
+import completeSound from '../sounds/complete.mp3'
+import successSound from '../sounds/success.mp3'
+import no from '../sounds/no.wav'
+import fonSound from '../sounds/fon.mp3'
+import failSound from '../sounds/fail.mp3'
+import board from './Board'
+import { Card } from './Card'
+
+console.log(board);
+
+export const Game = (props) => {
+  const [cards, setCards] = useState(board);
+  const [openedCards, setOpenedCards] = useState([]);
+  const [flippedCards, setFlippedCards] = useState([]);
+  const [defeat, setDefeat] = useState(false);
+  const [seconds, setSeconds] = useState(10);
+  const foo = useRef();
+
+  const soundsValue = localStorage.getItem('soundsValue');
+  const musicValue = localStorage.getItem('musicValue');
+
+  const [playCardFlip] = useSound(flipCardSound, {
+    volume: 0.005 * soundsValue,
+  });
+  const [playSuccess] = useSound(successSound, {
+    volume: 0.01 * soundsValue,
+  });
+  const [playNo] = useSound(no, {
+    volume: 0.01 * soundsValue,
+  });
+  const [playComplete] = useSound(completeSound, {
+    volume: 0.05 * soundsValue,
+  });
+  const [playFail] = useSound(failSound, {
+    volume: 0.01 * soundsValue,
+  });
+  const [playBg, {stop}] = useSound(fonSound, {
+      volume: 0.0005 * musicValue,
+    });  
+
+  function flipCard(id, value) {
+    playCardFlip();
+    const card = {
+      id: id,
+      value: value
+    }
```

Comment:
можно использовать сокращённую запись
```suggestion
    const card = {
      id,
      value
    }
```

или, ещё лучше, сохранять исходную карточку, всё равно названия ключей остаются прежние

Reply:
Да согласен, если названия ключей и значений совпадают то можно  не дублировать, но я так сделал и тогда значения не подсвечиваются, и я подумал от греха подальше лучше оставлю)
насчёт сохранения исходной карты не до конца понял как это реализовать и нужно ли это)